### PR TITLE
Fixed #71: guest names and ssh setup in Ansible provisioner

### LINKED ISSUE #71: [Bug] Ansible provisioner: wrong guest names and ssh not autostarted for certain guests
In [provisioners/ansible.py](https://github.com/lxdock/lxdock/blob/master/lxdock/provisioners/ansible.py):

1. The names of ArchLinux and OracleLinux should be changed to `arch` (L19) and `ol` (L25) instead of `archlinux` and `oracle`, respectively. The names are used for detecting the distribution ([see the code](https://github.com/lxdock/lxdock/blob/master/lxdock/guests/base.py#L87-L91)) and thus will stay as `arch` and `ol`.

2. On Arch, CentOS, Fedora, openSUSE and Oracle, it seemed that sshd was not started automatically. So we need the similar setup methods for them.

What I tried (needs to be verified further):

```python
    def setup_guest_arch(self):
        # On archlinux guests we have to ensure that sshd is started!
        # https://bbs.archlinux.org/viewtopic.php?id=205367
        self.guest.run(['systemctl', 'enable', 'sshd.socket', '--now'])

    def setup_guest_centos(self):
        # On centos guests we have to ensure that sshd is started!
        self.guest.run(['systemctl', 'enable', 'sshd.socket', '--now'])

    def setup_guest_fedora(self):
        # On fedora guests we have to ensure that sshd is started!
        self.guest.run(['systemctl', 'enable', 'sshd.socket', '--now'])

    def setup_guest_opensuse(self):
        # On opensuse guests we have to ensure that sshd is started!
        self.guest.run(['systemctl', 'enable', 'sshd'])

    def setup_guest_ol(self):
        # On oracle linux guests we have to ensure that sshd is started!
        self.guest.run(['systemctl', 'enable', 'sshd.socket', '--now'])
```

### PR DESCRIPTION
**Pull request by lingxiaoyang**
This pull request fixes guest names, package name, and commands ensuring sshd in Ansible provisioner. 

The only missing guest is openSUSE, which needs too much security configuration that is out of the scope of LXDock, in my opinion.

You may test with the following gist using `lxdock -v up`:
https://gist.github.com/lingxiaoyang/4ae30eed845fe0cccf444d82371abbff

### CODE REVIEW

**Comment by hsoft** [comment]
@ellmetha Is the codecov thing really useful to us? For me, it's just noise.

**Comment by ellmetha** [comment]
@hsoft I think it remains useful in order to browse code coverage online. That said pull request comments are a bit noisy IMO. I'll add a ``codecov.yml`` in order to try disabling them.

File: lxdock/provisioners/ansible.py [hunk 1]
```diff
@@ -16,13 +16,13 @@ class AnsibleProvisioner(Provisioner):
     name = 'ansible'
 
     guest_required_packages_alpine = ['openssh', 'python', ]
-    guest_required_packages_archlinux = ['openssh', 'python2', ]
+    guest_required_packages_arch = ['openssh', 'python', ]
```
Comment (hsoft) [inline lxdock/provisioners/ansible.py]: Under arch, `python` refers to python 3. Ansible requires (unless you're bleeding edge and want to use the newly added py3 support in ansible) python2.

**Comment by hsoft** [comment]
@lingxiaoyang to be sure to understand the problem you're fixing: under the 4 distros you're adding code for, sshd wasn't running without that?

File: lxdock/provisioners/ansible.py (hunk 1 above)
Comment (lingxiaoyang) [inline lxdock/provisioners/ansible.py]: @hsoft I didn't realize the Python versions support... python2 was installed as `python2` binary so I got an error of "Cannot find python". Perhaps I need a symlink, or if there's an Ansible configuration on which Python to use, I can also do it.

**Comment by lingxiaoyang** [comment]
@hsoft No, sshd was not running and I got a connection error when executing `ansible-playbook`.

File: lxdock/provisioners/ansible.py (hunk 1 above)
Comment (hsoft) [inline lxdock/provisioners/ansible.py]: Oh, that's true. On arch, python 2 has the `python2` executable name. That's a problem. We might as well leave thing as they are and let Ansible run under Python 3 on Arch guests. It *should* work most of the time...

**Review by ghost — APPROVED** [review]

**Pull request merged**